### PR TITLE
Corrections on Auth.js

### PR DESCRIPTION
**Pull request by jphNovitz**
This project was very useful but doesn't work for me.  I did some light changes for a good for me.  I hope this will help future users.

### CODE REVIEW

File: src/Auth.js [hunk 1]
```diff
@@ -2,12 +2,12 @@
 const Auth = {
 	install(vue, options = { loginUrl: "/api/login", signupUrl: "/api/users", logoutUrl: "/api/logout", refresh: false}) {
 		vue.prototype.$auth = new Authenticate(vue, options.loginUrl, options.signupUrl, options.logoutUrl);
-
 		vue.http.interceptors.push((request, next) => {
 
 	        if (!request.headers.hasOwnProperty('Authorization')) {
-	            request.headers['Authorization'] = localStorage.getItem("token");
-	        }
+                request.headers.set('Authorization', "BEARER "+localStorage.getItem("token"));
```
Comment (joostlawerman) [inline src/Auth.js]: Could you change it to `Bearer` ? It's more common

**Review by joostlawerman — CHANGES_REQUESTED** [review]
Awesome! Thank you for the additions. Could you take a look at my comment?

**Comment by jphNovitz** [comment]
I replace BEARER to Bearer 
It's better ... more common you're right

**Review by joostlawerman — APPROVED** [review]

**Comment by joostlawerman** [comment]
Hey @jphNovitz, I have released a beta version with your changes and some code style fixes you can pull it from the `beta` tag. (`[yarn|npm] [add|install] vue-token@beta`) the version should be `2.0.0-1`

**Pull request merged**